docs/semgrep-code/editor.md

```diff
@@ -215,12 +215,14 @@ To run your rule against selected repositories or projects:
 
 ## Set a rule’s visibility and share a rule
 
-Upon saving, a rule’s visibility is **unlisted** by default. This rule can be shared with anyone through an identifier, even to non-Semgrep AppSec Platform users.
+Upon saving, a rule’s visibility is **private** by default. A private rule is visible only to members within an organization.
 
-A rule can be saved as a **private rule**, which is visible only to members within an organization. You can still share a private rule, but only members of the organization can see it. Private rules are a **Team/Enterprise tier feature**.
+- To share a rule, click **Share > Public > Confirm**. If you want to share this specific version of the rule, you can also toggle Permalink. This provides a shortlink to this version of the rule, which will not change if the rule is modified.
```

Comment:
```suggestion
- To share a rule outside your organization, click **Share > Public > Confirm**. If you want to share this specific version of the rule, you can also toggle Permalink. This provides a shortlink to this version of the rule, which will not change if the rule is modified.
```

Reply:
this is covered in the next line, but never rely on someone reading two lines of docs 😂